lib/engine/step/g_1856/bankrupt.rb

```diff
@@ -0,0 +1,66 @@
+# frozen_string_literal: true
```

Comment:
Note to others; I have no clue what I'm doing here and I'll have to work on this another day to figure out how to make the bankruptcy work. In my test game when the player had no more cash or sellable shares to pay interest, it soft locked instead of offering bankruptcy.

Reply:
I'm going to leave proper bankrupcy handling for a future PR; for now the player can get to clicking on the bankrupt button and that is enough for now.